### PR TITLE
ADEN-5318 | Implement changes for video admixes

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-5330

* Disabled FMR on Featured Video pages in admix1
* limit FMR size on Featured Video pages in admix3

@Wikia/adeng

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 5 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adSizeFilter.js#L8](https://github.com/Wikia/app/blob/abece13b04c150205a13f24e482a5d07ae487d66/extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js#L8): Function has 5 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [adSizeFilter.js#L35](https://github.com/Wikia/app/blob/abece13b04c150205a13f24e482a5d07ae487d66/extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js#L35): Function has a complexity of 16 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [floatingMedrec.js#L10](https://github.com/Wikia/app/blob/abece13b04c150205a13f24e482a5d07ae487d66/extensions/wikia/AdEngine/js/slot/floatingMedrec.js#L10): Function has 7 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [floatingMedrec.js#L136](https://github.com/Wikia/app/blob/abece13b04c150205a13f24e482a5d07ae487d66/extensions/wikia/AdEngine/js/slot/floatingMedrec.js#L136): Function has a complexity of 11 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![INFO][INFO] [floatingMedrec.js#L50](https://github.com/Wikia/app/blob/abece13b04c150205a13f24e482a5d07ae487d66/extensions/wikia/AdEngine/js/slot/floatingMedrec.js#L50): Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/AdEngine/js/slot/floatingMedrec.js [hunk 1]
```diff
@@ -198,6 +198,11 @@ define('ext.wikia.adEngine.slot.floatingMedrec', [
 				return;
 			}
 
+			if (context.opts.adMix1Enabled && context.targeting.hasFeaturedVideo) {
```
Comment (fraszczakszymon) [inline extensions/wikia/AdEngine/js/slot/floatingMedrec.js]: Can we keep it here?
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/context/adContext.js#L102

**Comment by WikiaTech** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [adContext.js#L14](https://github.com/Wikia/app/blob/ea8ae5cc0a0711c4d5a45f30e82e01d4ecf86808/extensions/wikia/AdEngine/js/context/adContext.js#L14): Function has 8 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [adContext.js#L69](https://github.com/Wikia/app/blob/ea8ae5cc0a0711c4d5a45f30e82e01d4ecf86808/extensions/wikia/AdEngine/js/context/adContext.js#L69): Function has a complexity of 11 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [adContext.js#L117](https://github.com/Wikia/app/blob/ea8ae5cc0a0711c4d5a45f30e82e01d4ecf86808/extensions/wikia/AdEngine/js/context/adContext.js#L117): Function has a complexity of 31 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [adContext.js#L193](https://github.com/Wikia/app/blob/ea8ae5cc0a0711c4d5a45f30e82e01d4ecf86808/extensions/wikia/AdEngine/js/context/adContext.js#L193): Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1067)
1. ![MAJOR][MAJOR] [adSizeFilter.js#L8](https://github.com/Wikia/app/blob/ea8ae5cc0a0711c4d5a45f30e82e01d4ecf86808/extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js#L8): Function has 5 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [adSizeFilter.js#L35](https://github.com/Wikia/app/blob/ea8ae5cc0a0711c4d5a45f30e82e01d4ecf86808/extensions/wikia/AdEngine/js/provider/gpt/adSizeFilter.js#L35): Function has a complexity of 16 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [desktop.run.js#L21](https://github.com/Wikia/app/blob/ea8ae5cc0a0711c4d5a45f30e82e01d4ecf86808/extensions/wikia/AdEngine/js/run/desktop.run.js#L21): Function has 17 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [desktop.run.js#L94](https://github.com/Wikia/app/blob/ea8ae5cc0a0711c4d5a45f30e82e01d4ecf86808/extensions/wikia/AdEngine/js/run/desktop.run.js#L94): Function has 8 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by fraszczakszymon** [comment]
LGTM :+1:

**Pull request merged**